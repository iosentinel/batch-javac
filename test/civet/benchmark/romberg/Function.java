package benchmark.romberg;


public abstract class Function {
	abstract public float apply(float x);
}
