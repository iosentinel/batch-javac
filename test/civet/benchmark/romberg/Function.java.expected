package benchmark.romberg;

public abstract class Function {
    
    public Function() {
        super();
    }
    
    public abstract float apply(float x);
}